docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -351,11 +361,52 @@ The <PyObject module="dagster" object="io_manager" /> decorator behaves nearly i
 
 The provided `context` argument for `handle_output` is an <PyObject module="dagster" object="OutputContext" />. The provided `context` argument for `load_input` is an <PyObject module="dagster" object="InputContext" />. The linked API documentation lists all the fields that are available on these objects.
 
+
+## Defining filesystem-based IO Managers
```

Comment:
I think this would make most sense inside the "Examples" section.

Reply:
I moved the details about the `UPathIOManager` into this section. Do you still think it belongs to "Examples"?

Reply:
I still do think it belongs in examples.  The way I've thought about the structure of this page is that first we cover the general-purpose I/O manager functionality that everyone will need to understand. Then the examples section contains detail on more specific use cases.  I think it would make sense for it to go at the top of the examples section, because it's a fairly common specific use case.

Reply:
ok done